src/main/java/com/test/studentmanagement/entity/Department.java

```diff
@@ -0,0 +1,23 @@
+package com.test.studentmanagement.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "Departments")
+public class Department {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String departmentName;
```

Comment:
`@NotEmpty`

Reply:
I'm using Java 17, so it does not have the @NotEmpty annotation. Therefore, I have used nullable = false.